recipes/configs/gemma2/27B_full.yaml

```diff
@@ -0,0 +1,72 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a gemma2 27B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download google/gemma-2-27b --ignore-patterns "gemma-2-27b.gguf"  --hf-token <HF_TOKEN>
+#
+# To launch on 4 devices, run the following command from root:
+#   tune run --nnodes 1 --nproc_per_node 4 full_finetune_distributed --config gemma2/27B_full
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training
+# you can run:
+#   tune run --nnodes 1 --nproc_per_node 4 full_finetune_distributed --config gemma2/27B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works only when the model is being fine-tuned on 2+ GPUs.
+
+
+# Tokenizer
+tokenizer:
+  _component_: torchtune.models.gemma.gemma_tokenizer
+  path: /tmp/gemma-2-27b/tokenizer.model
+
+# Dataset
+dataset:
```

Comment:
Sorry to potentially be a pain in the ass here. We have parallel PR (https://github.com/pytorch/torchtune/pull/1872) which is helping standardize our configs and better expose the features we have. This means we always have `packed: False` in `dataset`, and `log_peak_memory_stats: True` and `compile: False` below, for every one of our configs.

Would it be annoying to ask if we could update these in the same way while we're here, please?

Reply:
Done I have updated all the configs to match the other PR!